pkg/initialize/run.go

```diff
@@ -61,6 +61,12 @@ func DownloadPolicies() error {
 
 	tempPath := filepath.Join(os.TempDir(), "terrascan")
```

Comment:
Hey @dev-gaur,
Would it make sense to use `ioutil.TempDir()` instead of `os.TempDir()`

Reply:
eureka!

Reply:
changes made. PTAL again